src/foreign/print_helper/etable_op_configure.rs

```diff
@@ -0,0 +1,136 @@
+use crate::{
+    circuits::{
+        etable_compact::{
+            op_configure::{
+                ConstraintBuilder, EventTableCellAllocator, EventTableOpcodeConfig,
+                MTableLookupCell, U64Cell,
+            },
+            EventTableCommonConfig, MLookupItem, StepStatus,
+        },
+        mtable_compact::encode::MemoryTableLookupEncode,
+        utils::{bn_to_field, Context},
+    },
+    constant, constant_from,
+    foreign::{EventTableForeignCallConfigBuilder, ForeignCallInfo},
+};
+use halo2_proofs::{
+    arithmetic::FieldExt,
+    plonk::{Error, Expression, VirtualCells},
+};
+use num_bigint::BigUint;
+use specs::step::StepInfo;
+use specs::{
+    etable::EventTableEntry,
+    itable::{OpcodeClass, OPCODE_CLASS_SHIFT},
+};
+use specs::{host_function::HostPlugin, mtable::VarType};
+
+pub struct PrintForeignCallInfo {}
+impl ForeignCallInfo for PrintForeignCallInfo {
+    fn call_id(&self) -> usize {
+        OpcodeClass::ForeignPluginStart as usize + HostPlugin::Print as usize
+    }
+}
+
+pub struct ETablePrintHelperTableConfig {
+    value: U64Cell,
+    value_lookup: MTableLookupCell,
+}
+
+pub struct ETablePrintHelperTableConfigBuilder {}
+
+impl<F: FieldExt> EventTableForeignCallConfigBuilder<F> for ETablePrintHelperTableConfigBuilder {
+    fn configure(
+        common: &mut EventTableCellAllocator<F>,
+        _constraint_builder: &mut ConstraintBuilder<F>,
+        _info: &impl ForeignCallInfo,
+    ) -> Box<dyn EventTableOpcodeConfig<F>> {
+        let value = common.alloc_u64();
```

Comment:
Since it has no constraints, we can ignore the value and its lookup. Only constraint the sp change.